src/components/ad-service-provider.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { DfpAdWrapper } from "./dfp-ad-wrapper";
+
+export function AdServiceProvider(props) {
+  const { adService = "", defaultNetworkID = "" } = props;
+
+  if(!defaultNetworkID) {
+    return null;
+  }
+
+  switch(adService) {
+    case "dfp":
+      return <DfpAdWrapper {...props} />;
+    default:
+      return <DfpAdWrapper {...props} />;
```

Comment:
I assume this is an API proposal? Because else just does DFP again

Reply:
yes, right.